acceptance/Gemfile

```diff
@@ -9,5 +9,5 @@ gem "inspec"
 # puts in a box_url for bento when a vagrant box in atlas is specified
 gem "kitchen-vagrant", github: "test-kitchen/kitchen-vagrant"
 gem "windows_chef_zero"
-gem "winrm-fs", "~> 0.4.0"
+gem "winrm-fs"
```

Comment:
Test Kitchen [pins](https://github.com/test-kitchen/test-kitchen/blob/637ec6693a618272d9f6a5291373facaf32979f1/test-kitchen.gemspec#L37) this now